docs/source/module_configuration/staticsite.rst

```diff
@@ -43,71 +305,62 @@ Incorporating CloudFront with a static site is a common best practice, however,
-              # pre-built function versions
-              # (this takes precedence over staticsite_rewrite_directory_index)
-              staticsite_lambda_function_associations:
-                - type: origin-request
-                  arn: arn:aws:lambda:us-east-1:123456789012:function:foo:1
+  deployments:
+    - modules:
+      - path: sample-app
+        type: static
+        parameters:
+          dev:
+            namespace: sample-app-dev
+            staticsite_auth_at_edge: true
+            staticsite_user_pool_arn: arn:aws:cognito-idp:us-east-1:240134083525:userpool/us-east-1_cjVgcUyWV
+      regions:
+        # NOTE: At the time of this writing Lambda@EDGE is only
+        # available in us-east-1. This solution will fail in
+        # other regions
+        - us-east-1
 
-              # Custom error response options can be defined
-              staticsite_custom_error_responses:
-                - ErrorCode: 404
-                  ResponseCode: 200
-                  ResponsePagePath: /index.html
+The `Auth@Edge`_ functionality does not create a Cognito User Pool for you.
+This is a manual step that needs to be completed prior to launching
+the stack as a user pool arn is required.  A client will be automatically
+created within this user pool for use with the application.
 
-              # Don't use CloudFront with the site
-              # i.e. for a development site accessible only via its S3-url
-              statisite_cf_disable: true
-            options:
-              pre_build_steps:  # commands to run before generating hash of files
-                - command: npm ci
-                  cwd: ../myothermodule  # directory relative to top-level path setting
-                - command: npm run export
-                  cwd: ../myothermodule
-              source_hashing:  # overrides for source hash collection/tracking
-                enabled: true  # if false, build & upload will occur on every deploy
-                parameter: /${namespace}/myparam  # defaults to <namespace>-<name/path>-hash
-                directories:  # overrides default hash directory of top-level path setting
-                  - path: ./
-                  - path: ../common
-                    # Additional (gitignore-format) exclusions to hashing
-                    # (.gitignore files are loaded automatically)
-                    exclusions:
-                      - foo/*
-              build_steps:
-                - npm ci
-                - npm run build
-              build_output: dist  # overrides default directory of top-level path setting
-        regions:
-          - us-west-2
+Federated Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Auth@Edge`_ does support federated identities for it's authentication protocol (Google, Facebook, OneLogin, ect.). A limitation to this is you can only have Federated login types or Cognito login types, not both. This will be addressed in a future release.
```

Comment:
I'm so glad someone else did the legwork to find a workaround for this -- should be able to fix by incorporating https://github.com/aws-samples/cloudfront-authorization-at-edge/pull/32

Reply:
Implemented